packages/app/src/client/integrations/k8sMetrics/Show/InstallInstructions.tsx

```diff
@@ -89,13 +117,45 @@ export const InstallInstructions = ({
   };
 
   const dashboardHandler = async () => {
-    const folder = await grafana.createFolder({ integration, tenant });
+    let folder;
+    try {
+      folder = await grafana.createFolder({ integration, tenant });
+    } catch (error) {
+      notifyError(
+        `Could not create grafana integration dashboard folder ${integration.name}`,
+        error.response.data.message ?? error.message
+      );
+      return;
+    }
 
     for (const d of makePrometheusDashboardRequests({
       integrationId: integration.id,
       folderId: folder.id
     })) {
-      await grafana.createDashboard(tenant, d);
+      try {
+        await grafana.createDashboard(tenant, d);
+      } catch (error) {
+        /*
+          Errors are usually communicated as a JSON in a list. 
+          For example:
+          [
+            { "fieldNames": [ "Dashboard" ], "classification": "RequiredError", "message": "Required" }
+          ] 
+          There might be ways to derrive better, human readable error messages, but for now we're 
+          just showing the error JSON.
+         */
+        const errorMessageList = Array.isArray(error.response.data)
+          ? error.response.data.map((errObj: Object) => JSON.stringify(errObj))
+          : [error.message];
+
+        for (const message of errorMessageList) {
+          notifyError(
+            `Could not create grafana integration dashboard ${d.uid}`,
+            message
+          );
```

Comment:
Real world example
![Screenshot 2021-07-29 at 17 10 30](https://user-images.githubusercontent.com/64152453/127519816-4d17886d-3209-470d-95a9-884de1a31042.png)

Reply:
Not the best error message, but might be understandable.